### PR TITLE
Test Tanner connection on startup

### PR DESCRIPTION
**Pull request by afeena**
- run version checker in Pool executor 
- check TANNER availability

#31

### CODE REVIEW

File: snare.py [hunk 1]
```diff
@@ -369,20 +372,16 @@ def parse_timeout(timeout):
...
-    print('serving on {0} with uuid {1}'.format(srv.sockets[0].getsockname()[:2], snare_uuid.decode('utf-8')))
-    try:
-        loop.run_forever()
-    except (KeyboardInterrupt, TypeError) as e:
-        print(e)
+@asyncio.coroutine
+def check_tanner_connection():
+    with aiohttp.ClientSession() as client:
+        req_url = 'http://{}:8090'.format(args.tanner)
+        print(req_url)
```
Comment (glaslos) [inline snare.py]: remove this print

File: snare.py [hunk 2]
```diff
@@ -369,20 +372,16 @@ def parse_timeout(timeout):
...
+        try:
+            resp = yield from client.get(req_url)
+        except aiohttp.errors.ClientOSError:
+            print("Can not connect to tanner host {}".format(req_url))
```
Comment (glaslos) [inline snare.py]: `Can't` :)

File: snare.py [hunk 3]
```diff
@@ -395,6 +394,7 @@ def server():
 
     """)
     snare_uuid = snare_setup()
+    check_tanner_connection()
```
Comment (glaslos) [inline snare.py]: could we move this before the `snare_setup()`?

File: snare.py [hunk 4]
```diff
@@ -403,7 +403,7 @@ def server():
     parser.add_argument("--port", help="port to listen on", default='8080')
     parser.add_argument("--interface", help="interface to bind to", default='localhost')
     parser.add_argument("--debug", help="run web server in debug mode", default=False)
-    parser.add_argument("--tanner", help="ip of the tanner service", default='tanner.mushmush.org')
+    parser.add_argument("--tanner", help="ip of the tanner service")
```
Comment (glaslos) [inline snare.py]: Why no default?

Comment (afeena) [inline snare.py]: Ooops, It's artifacts after testing, I think

**Comment by afeena** [comment]
I fixed problems and made rebase.

**Pull request merged**